packages/vusd-lib/src/utils.js

```diff
@@ -1,12 +1,20 @@
 'use strict'
 
-const Big = require('big.js').default
+const Big = require('big.js')
 
+/**
+ * @param {string} number
+ * @param {number} decimals
+ */
 const fromUnit = (number, decimals = 18) =>
-  Big(`${Big(number).toFixed()}e-${decimals}`).toFixed()
+  new Big(`${new Big(number).toFixed()}e-${decimals}`).toFixed()
 
+/**
+ * @param {string} number
+ * @param {number} decimals
+ */
 const toUnit = (number, decimals = 18) =>
-  Big(`${Big(number).toFixed()}e+${decimals}`).toFixed(0)
+  new Big(`${new Big(number).toFixed()}e+${decimals}`).toFixed(0)
```

Comment:
A minor comment, but `Big` does not require new. So you can update the ESLint rule with

```json
 "new-cap": [
  "warn",
  {
     "capIsNewExceptions": ["Big"]
   }
],
```

and remove all the extra `new`s. We've done so in other projects

Reply:
I find weird to use `Big` without `new` as the convention is to use capitalize constructors, which must use `new`. `Big` allows that dual usage but I prefer to consistently use `new` over a shorter code.